### DISCUSSION TITLE
Cryptgeon: A short guide to self-hosting - An excellent service for secure data and file transfer - Transfer data and files securely and encrypted

### DISCUSSION THREAD

**Discussion by nicfab** [cupcakearmy/cryptgeon#141] [General]
Kudos, @cupcakearmy, for this project!

You can find my article on **CryptGeon** here:

**IT**: https://notes.nicfab.eu/it/posts/cryptgeon/

**EN**: https://notes.nicfab.eu/en/posts/cryptgeon/

Also on social platforms:
- Mastodon: https://fosstodon.org/@nicfab/112744302473665146
- Bluesky: https://bsky.app/profile/nicfab.bsky.social/post/3kwolkapwcw2b
- Hacker News: https://news.ycombinator.com/item?id=40896148
- X: https://twitter.com/nicfab/status/1809872724036047121?s=61&t=Pm0VJo0MKSJVU2uGgulm9A
- LinkedIn: https://www.linkedin.com/posts/nicfab_cryptgeon-a-short-guide-to-self-hosting-activity-7215636966069944320-Cf0P?utm_source=share&utm_medium=member_desktop

All the best!

**Comment by cupcakearmy** [comment]
Grazie mille ❤️
